web/src/app/rotations/RotationAddUserDialog.js

```diff
@@ -12,66 +11,54 @@ const mutation = gql`
   }
 `
 
-export default class RotationAddUserDialog extends React.Component {
-  static propTypes = {
-    rotationID: p.string.isRequired,
-    userIDs: p.array.isRequired,
-    onClose: p.func.isRequired,
+const RotationAddUserDialog = (props) => {
+  const { rotationID, onClose } = props
+  const [value, setValue] = useState(null)
+  const defaultValue = {
+    users: [],
   }
+  // append to users array from selected users
+  const users = []
+  const userIDs = (value && value.users) || defaultValue.users
+  props.userIDs.forEach((u) => users.push(u))
```

Comment:
Let's call `userIDs` flat (destructered from props like the other prop variables on [L15](https://github.com/target/goalert/pull/1452/files?diff=split&w=1#diff-4fe778655b2db0c3985ad8d1b6c82ffd2a29d801955e9fc7c9f35720f2ea42faR15)), and rename the local variable to something like `uids`.